src/kudu/consensus/consensus_queue.cc

```diff
@@ -382,21 +383,31 @@ unordered_map<string, HealthReportPB> PeerMessageQueue::ReportHealthOfPeers() co
   return reports;
 }
 
-void PeerMessageQueue::CheckPeersInActiveConfigIfLeaderUnlocked() const {
+void PeerMessageQueue::UpdatePeersInActiveConfigIfLeaderUnlocked() {
```

Comment:
As discussed in 1:!, Why are we updating peer quorum id each time SetLeaderMode is called or TrackedPeer handling is happening. setting of quorum id seems unrelated to the original intent of function CheckPeersInActiveConfigIfLeaderUnlocked